apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -314,41 +316,41 @@ test_func func_table[WM_TEST_MAX] = {
 exec_func exec_table[WM_TEST_MAX] = {
 	NULL,                                      /* WM_TEST_START    */
 	NULL,                                      /* WM_TEST_STOP     */
-	_wm_test_softap,                           /* WM_TEST_SOFTAP   */
+	_wm_test_softap,                         /* WM_TEST_SOFTAP   */
 	NULL,                                      /* WM_TEST_STA      */
-	_wm_test_join,                             /* WM_TEST_JOIN     */
+	_wm_test_join,                           /* WM_TEST_JOIN     */
 	NULL,                                      /* WM_TEST_LEAVE    */
 	NULL,                                      /* WM_TEST_CANCEL   */
-	_wm_test_set,                              /* WM_TEST_SET      */
+	_wm_test_set,                            /* WM_TEST_SET      */
 	NULL,                                      /* WM_TEST_GET      */
 	NULL,                                      /* WM_TEST_RESET    */
-	_wm_test_scan,                             /* WM_TEST_SCAN     */
+	_wm_test_scan,                           /* WM_TEST_SCAN     */
 	NULL,                                      /* WM_TEST_MODE     */
 	NULL,                                      /* WM_TEST_STATS    */
 	NULL,                                      /* WM_TEST_INFO     */
-	_wm_test_auto,                             /* WM_TEST_AUTO     */
+	_wm_test_auto,                           /* WM_TEST_AUTO     */
 	_wm_test_join,
-	_wm_test_stress,                                       /* WM_TEST_STRESS   */
+	_wm_test_stress,                         /* WM_TEST_STRESS   */
 };
 
 char *func_name[WM_TEST_MAX] = {
-	"start",
-	"stop",
-	"softap",
-	"sta",
-	"join",
-	"leave",
-	"cancel",
-	"set",
-	"get",
-	"reset",
-	"scan",
-	"mode",
-	"stats",
-	"info",
-	"auto",
+	"start",                                /* WM_TEST_START    */
+	"stop",								    /* WM_TEST_STOP     */
+	"softap",							    /* WM_TEST_SOFTAP  */
+	"sta",								    /* WM_TEST_STA      */
+	"join",								    /* WM_TEST_JOIN    */
+	"leave",							    /* WM_TEST_LEAVE    */
+	"cancel",							    /* WM_TEST_CANCEL   */
```

Comment:
Can we please confirm whether comments are aligned?

Reply:
aligning the comments is different to each editor. So I don't want fix it